db/migrate/20200421105454_create_complex_cases.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComplexCases < ActiveRecord::Migration[5.2]
+  def change
+    create_table :complex_cases, id: :uuid do |t|
+      t.string :key, null: false
```

Comment:
might be good to add an index on `key`

Reply:
There are only 4 rows, and I don't think I'll be joining against key in the immediate future so not sure it needs an index right now.

Reply:
👍